deployment/src/test/java/io/quarkiverse/loggingmanager/quarkus/logging/manager/deployment/AbstractConfigurationTest.java

```diff
@@ -1,4 +1,4 @@
-package io.quarkiverse.loggingui.quarkus.logging.ui.deployment;
+package io.quarkiverse.loggingmanager.quarkus.logging.manager.deployment;
```

Comment:
package name still the long one. Can be changed to `io.quarkiverse.loggingmanager..deployment`